boards/airfy-beacon/include/board.h

```diff
@@ -28,7 +28,7 @@
 #endif
 
 /**
- * @brief   Xtimer configuration
+ * @name   Xtimer configuration
```

Comment:
Shouldn't be 's/@brief/@name /g' ;-). It's not mandatory, but the next person trying to add some doc might be happy about fitting alignment.